src/table.py

```diff
@@ -1365,6 +1365,46 @@ def char_in_bbox(char, bbox) -> bool:
 
         return table_arr
 
+    def to_markdown(self, clean=True):
+        """Output table content as a string in Github-markdown format.
+
+        If clean is true, markdown syntax is removed from cell content."""
+        output = "|"
+
+        # generate header string and MD underline
+        for i, name in enumerate(self.header.names):
+            if name is None or name == "":  # generate a name if empty
+                name = f"Col{i+1}"
+            name = name.replace("\n", " ")  # remove any line breaks
+            if clean:  # remove sensitive syntax
+                name = (
+                    name.replace("<", "&lt;").replace(">", "&gt;").replace("-", "&#45;")
```

Comment:
`html.escape` might be better here and elsewhere? Also escapes `&`, and doesn't escape `-` which seems better?

Reply:
where do I find definition of `html.escape`?